app/views/shelter_reviews/edit.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Edit Review</h1>
+
+<% flash.each do |type, message| %>
+  <%= message %>
+<% end %>
```

Comment:
Displays a flash message when user does not enter required fields. Very plain right now but would eventually like to style with bootstrap.

Reply:
Yeah, we should agree on a convention/style for displaying the flash message within the project. We can discuss this one later.